docs/docs/quickstart.md

```diff
@@ -1,9 +1,30 @@
 # Quick Start
 
-**Note**: To run with 2.12 please make sure that the flag `-Ypartial-unification`
+## Dependencies
+
+http4s is available for Scala 2.12, 2.13, and 3.2
+
+You can add http4s to your build by adding its modules to your dependencies in `build.sbt`:
+
+```scala
+libraryDependencies ++= Seq(
+  "org.http4s" %% "http4s-core"         % @VERSION@,
+  "org.http4s" %% "http4s-client"       % @VERSION@,
+  "org.http4s" %% "http4s-server"       % @VERSION@,
```

Comment:
Not sure that end-users are interested to have those explicit dependencies, honestly 🤔

Reply:
Just to make sure I'm clear, you're thinking we don't need `http4s-client` and `http4s-server`?
And perhaps that the ember ones would be enough?

Reply:
If we speak about regular users, not some library developers, I think yes. They will need only ember/blaze/http4s-netty dependency + dsl maybe.

Reply:
But yeah this question is tricky enough. http4s has nice granularity, and you never know the needs of a particular user. So maybe it's better to add several examples, like 'add these dependencies if you need http client/server to go' and the counterpart one for library developers.

Reply:
Yeah that's fair.
I think the common use case is mostly that you've forgotten the group ID and just copy paste what you need.
Not that this is a list that you should add exactly. Although it's not crazy if they do.

Reply:
Yeah, exactly. I also feel that newcomers would copy the whole suggested dependencies to their builds without paying extra attention. And to be fair, that's a fine-working approach most of the time 😄

Reply:
...wait, did we end up on "this list is actually ok" or should we add / remove something?

Reply:
I wonder if instead of listing all the deps, first step should be "choose an integration" from this page, since that will get you precisely the right deps.
https://http4s.org/v0.23/docs/integrations.html

I've been dreaming recently if `http4s-client` just included ember client, alas ...

Reply:
> first step should be "choose an integration" from this page

I do specifically want to avoid any other first step here.
From the motivating Discord message:
> Could you please place a code snippet for sbt in the official documentation, on a page that can be reached immediately from the landing page?
> I have to search for the version and organization name every time I use this library.

So I've opted to add a link to the integrations page for more modules.

![image](https://user-images.githubusercontent.com/5440389/217586506-0b9cb1be-d077-4d69-b67b-2a0f89c798eb.png)

Reply:
It is all the matter of personal perception, of course, but from my prospective, in most cases users will be referencing to the `http4s-ember-client` and `http4s-ember-server` dependencies somewhere at the code intialization / dependency injection point only. Most of the user's code is supposed to be using definitions from `http4s-core/-server/-client`. So to me it looks like a good idea to have these dependencies referenced explicitly in their project files along with the `-ember-` ones. Just because the core dependencies are "working horses" in code in most cases, actually. So it might not be that bad to let users be familiar with them from the very beginning.

Reply:
> In that case, maybe we should specifically put the ember deps + dsl first, in its own code block, since this is probably what users want. Then we can list other deps and link to integrations.

💯 that's what I meant precisely.

Reply:
> So it might not be that bad to let users be familiar with them from the very beginning.

Yeah, agree that it is better to know about these. I guess I have a couple fears (not backed by any hard evidence):
1. A user just wants a client, doesn't know what ember is, so they go for `http4s-client` and then are confused. This is pretty similar to getting `cats-effect-{kernel,std}` when what you really want is `IO`.
2. More dependencies in their `build.sbt` means more version numbers that can get out of sync, and because of internal package-private stuff this can breaking things in weird ways. Although I suppose we can mitigate this by showing that a common version number should be stored in a `val`.

Reply:
@satorg note that we are on the 'quickstart' page. So I'd guess it's really about something to grab rapidly and run away.

Reply:
Thanks everyone for all the input, I really appreciate it :)

I've switched the a `http4sVersion` val to help make clear these should all use the same version.
And I've bumped up the ember and dsl deps with a comment about getting up and running.
Added a small comment attempting to motivate why you might want the core classes.

![image](https://user-images.githubusercontent.com/5440389/217617727-ebc5a376-1648-43ab-9064-ff3c68a00d5f.png)

I want to avoid multiple code blocks.
Let's keep a couple things in mind, this is the 'Quick Start' page, we should keep ceremony to a minimum.
There is still the giter8 template for a more thorough example.

Reply:
> Added a small comment attempting to motivate why you might want the core classes.

Sorry to nitpick, I think it's still not entirely clear. It almost seems like you _won't_ get the core classes unless you add those deps, which is not actually the case.

The CE docs I linked make the distinction between application authors, and library/middleware authors, and that IMHO is the right distinction.

Reply:
Hmmm, how about?

```scala
val http4sVersion = "@VERSION@"

libraryDependencies ++= Seq(
  // get up and running with a client and server
  "org.http4s" %% "http4s-ember-client" % http4sVersion,
  "org.http4s" %% "http4s-ember-server" % http4sVersion,
  "org.http4s" %% "http4s-dsl"          % http4sVersion,

  // library code might need just the core classes and traits
  // e.g. `Uri`, `Client[F]`, `Server[F]`
  "org.http4s" %% "http4s-core"         % http4sVersion,
  "org.http4s" %% "http4s-client"       % http4sVersion,
  "org.http4s" %% "http4s-server"       % http4sVersion,
)
```

Reply:
I still think it's better done in its own code block, with explanation in the text instead of `//` comments. But I can propose that as a follow-up PR :)